dto/scenario_testrun.go

```diff
@@ -6,14 +6,20 @@ import (
 	"github.com/checkmarble/marble-backend/models"
 )
 
+// type CreateScenarioTestRunBody struct {
+// 	ScenarioIterationId string `json:"scenario_iteration_id"`
+// 	ScenarioId          string `json:"scenario_id"`
+// 	RefIterationId      string `json:"ref_iteration_id"`
+// 	PhantomIterationId  string `json:"phantom_iteration_id"`
+// 	StartDate           string `json:"start_date"`
+// 	EndDate             string `json:"end_date"`
+// 	Period              string `json:"period"`
+// }
+
 type CreateScenarioTestRunBody struct {
-	ScenarioIterationId string `json:"scenario_iteration_id"`
-	ScenarioId          string `json:"scenario_id"`
-	RefIterationId      string `json:"ref_iteration_id"`
-	PhantomIterationId  string `json:"phantom_iteration_id"`
-	StartDate           string `json:"start_date"`
-	EndDate             string `json:"end_date"`
-	Period              string `json:"period"`
+	PhantomIterationId string `json:"test_iteration_id"`
```

Comment:
this is just a lint blocker, you can ignore it if you want (but in the end I think it would make sense to align the model and the DTO)